app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/EateryCard.kt

```diff
@@ -247,7 +257,7 @@ fun EateryCard(
                             Image(
                                 bitmap = apiResponse.data,
                                 modifier = Modifier
-                                    .height(130.dp)
+                                    .height(if (isGridView) 100.dp else 130.dp)
```

Comment:
Lets take this out and put it in a variable so we don't have 3 points of potential error for typos